engine/src/flutter/testing/run_tests.py

```diff
@@ -1372,13 +1372,7 @@ def main():
     run_benchmark_tests(build_dir)
     run_engine_benchmarks(build_dir, engine_filter)
 
-  variants_to_skip = ['host_release', 'host_profile']
-
-  def should_skip(variant):
-    matches = [variant for variant in variants_to_skip if variant in args.variant]
-    return len(matches) > 0
-
-  if ('engine' in types or 'font-subset' in types) and not should_skip(args.variant):
+  if ('engine' in types or 'font-subset' in types) and 'debug' in build_dir:
```

Comment:
This looks like a half-finished migration from using `engine` to run these tests to using `font-subset` to run them. Instead of using the filters on the config name, we should just use the `font-subset` type. Doesn't have to be in this PR though.

Reply:
Filed https://github.com/flutter/flutter/issues/161538 to complete this.